src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -0,0 +1,1665 @@
+            (state.dataLoopNodes->Node(this->EvapOutletNodeNum).TempSetPointHi == DataLoopNode::SensedNodeFlagValue)) {
+            if (!state.dataGlobal->AnyEnergyManagementSystemInModel) {
+                if (!this->ModulatedFlowErrDone) {
+                    ShowWarningError(state, format("Missing temperature setpoint for LeavingSetpointModulated mode chiller named {}", this->Name));
+                    ShowContinueError(
+                        state, "  A temperature setpoint is needed at the outlet node of a chiller in variable flow mode, use a SetpointManager");
+                    ShowContinueError(state, "  The overall loop setpoint will be assumed for chiller. The simulation continues ... ");
+                    this->ModulatedFlowErrDone = true;
+                }
+            } else {
+                // need call to EMS to check node
+                bool fatalError = false; // but not really fatal yet, but should be.
+                EMSManager::CheckIfNodeSetPointManagedByEMS(
+                    state, this->EvapOutletNodeNum, EMSManager::SPControlType::TemperatureSetPoint, fatalError);
+                state.dataLoopNodes->NodeSetpointCheck(this->EvapOutletNodeNum).needsSetpointChecking = false;
+                if (fatalError) {
+                    if (!this->ModulatedFlowErrDone) {
+                        ShowWarningError(state,
+                                         format("Missing temperature setpoint for LeavingSetpointModulated mode chiller named {}", this->Name));
+                        ShowContinueError(state,
+                                          "  A temperature setpoint is needed at the outlet node of a chiller evaporator in variable flow mode");
+                        ShowContinueError(state, "  use a Setpoint Manager to establish a setpoint at the chiller evaporator outlet node ");
+                        ShowContinueError(state, "  or use an EMS actuator to establish a setpoint at the outlet node ");
+                        ShowContinueError(state, "  The overall loop setpoint will be assumed for chiller. The simulation continues ... ");
+                        this->ModulatedFlowErrDone = true;
+                    }
+                }
+            }
+            this->ModulatedFlowSetToLoop = true;
+            state.dataLoopNodes->Node(this->EvapOutletNodeNum).TempSetPoint =
+                state.dataLoopNodes->Node(state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).TempSetPointNodeNum).TempSetPoint;
+            state.dataLoopNodes->Node(this->EvapOutletNodeNum).TempSetPointHi =
+                state.dataLoopNodes->Node(state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).TempSetPointNodeNum).TempSetPointHi;
+        }
+    }
+}
+
+void ASHRAE205ChillerSpecs::initialize(EnergyPlusData &state, bool const RunFlag, Real64 const MyLoad)
+{
+    static constexpr std::string_view RoutineName("ASHRAE205ChillerSpecs::initialize");
+
+    if (this->oneTimeFlag) {
```

Comment:
This block shouldn't be in here.  The `setOutputVariables` function should be called from the `oneTimeInit` function, and you should eliminate the `->oneTimeFlag` variable and check entirely.  The point of the `oneTimeInit` was to eliminate this type check, and replace it with an intentional one-time pass over all the plant components [here](https://github.com/NREL/EnergyPlus/blob/a1211bbb5613f31be3bc6b789d48652b956b8005/src/EnergyPlus/Plant/PlantManager.cc#L1818-L1835), right after the factory methods are called.  I think this change should be made before merging.

Reply:
(please address this one)

Reply:
Fixed. 
So the way we had it, oneTimeInit was probably being called two times? 
A bit of boilerplate component code in the wiki might be helpful for this - if an unsuspecting dev copies and pastes from an existing file, the error gets propagated up to the reviewer every time. :(